vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -807,6 +807,8 @@
     <PackageReference Include="Microsoft.Windows.CppWinRT" Version="$(CppWinRTVersion)" />
     <PackageReference Include="ReactNative.Hermes.Windows" Version="$(HermesVersion)" />
     <PackageReference Include="$(WinUIPackageName)" Version="$(WinUIPackageVersion)" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22000.194" />
```

Comment:
Just want to double-down on a previous question. Is this the appropriate Windows SDK version (the default version installed by VS is lower).